internal/provider/resource_k8s.go

```diff
@@ -409,15 +444,19 @@ func (k *K8sNodeData) Dictify() map[string]interface{} {
 func (k *K8sDeployer) invalidateBrokenAttributes(sub *substrate.Substrate) error {
 	newWorkers := make([]K8sNodeData, 0)
 	validNodes := make(map[uint32]struct{})
```

Comment:
very neat way to use maps as sets

Reply:
golang doesn't have sets, event the set in collections pkgs uses map as a set